frontend/packages/ceph-storage-plugin/src/utils/dashboard.ts

```diff
@@ -1,4 +1,3 @@
-import * as _ from 'lodash';
 import { PrometheusResponse } from '@console/internal/components/graphs';
 
 export const getResiliencyProgress = (results: PrometheusResponse): number => {
```

Comment:
```suggestion
export const getResiliencyProgress = (results: PrometheusResponse = {}): number => {
```

Reply:
@afreen23 Same! We can't keep an empty object.

Reply:
I think here doing this would be fine:
```
export const getResiliencyProgress = (
  results: PrometheusResponse['data']['result'] = [],
): 
```